src/App.jsx

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 import { CssBaseline, ThemeProvider } from '@material-ui/core';
 import Footer from './components/Footer';
 import Header from './components/Header';
-import Home from './pages/Home';
+import Home from './pages/HomeSelection';
```

Comment:
The naming is inconsistent. How about we rename the existing `Home.jsx` to `PublicHome.jsx` and `HomeSelection` to `Home`?

Reply:
Yeah that could work as well, ill do that